src/vnsw/agent/vrouter/ksync/vnswif_listener_base.h

```diff
@@ -159,7 +159,7 @@ class VnswInterfaceListenerBase {
     uint8_t tx_buf_[kMaxBufferSize];
 
     int sock_fd_;
-    //WINDOWS-TEMP local::datagram_protocol::socket sock_;
```

Comment:
I don't see any line in this diff that started using this member. Assumming it was compiling before and after, I don't really get it.

Reply:
This file isn't even included now on Windows. Somebody commented it out and I recovered it (because this is a cleanup pull request).